src/Form/Mixin/HasLabel.php

```
@@ -0,0 +1,201 @@
+     *     },
+     * );
+     * $element->labelClass('another-class', true);
+     * $element->labelClass(null);
+     * ```
+     *
+     * @param string|Stringable|UnitEnum|null $value CSS class value, or `null` to remove the attribute.
+     * @param bool $override Whether to override existing classes (`true`) or merge (`false`).
+     *
+     * @return static New instance with the updated `labelAttributes['class']` value.
+     */
+    public function labelClass(string|Stringable|UnitEnum|null $value, bool $override = false): static
```

PHPMD (2.15.0)

[error] The method labelClass has a boolean flag argument $override, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)